apps/spruce/src/components/Content/index.tsx

```diff
@@ -52,6 +53,9 @@ export const Content: React.FC = () => (
       />
       <Route path={routes.host} element={<Host />} />
       <Route path={routes.hosts} element={<Hosts />} />
+      <Route path={`${routes.image}/*`} element={<Image />}>
+        <Route path={`:${slugs.tab}`} element={null} />
+      </Route>
```

Comment:
We can do something neat where we can hide the path on the production environment (but it will be accessible on development & staging). It's helpful to do this because the page isn't ready for production release yet!
```suggestion
      {!isProduction() && (<Route path={`${routes.image}/*`} element={<Image />}>
        <Route path={`:${slugs.tab}`} element={null} />
      </Route>)}
```

Use `import { isProduction } from "utils/environmentVariables";` at the top of the file to get access to the `isProduction()` function